cycode/cli/consts.py

```diff
@@ -189,3 +189,5 @@
 # Example: A -> B -> C
 # Result: A -> ... -> C
 SCA_SHORTCUT_DEPENDENCY_PATHS = 2
+
+SCA_DISABLE_RESTORE_DEPENDENCIES_FLAG = 'no-restore'
```

Comment:
Please insert sca-no-restore

Reply:
the name of the flag has been reviewed with product

Reply:
@morsa4406 we already have SCA only options without SCA prefix

like this:
>  `--severity-threshold [INFO\|LOW\|MEDIUM\|HIGH\|CRITICAL]`  | Show only violations at the specified level or higher (supported for the SCA scan type only).

WDYT?